about: merge use case pages (#80)
osrd-project/osrd-website: content/about/use-case/_index.en.md

---
title: Use cases
linkTitle: Use cases
description: Use cases related to rail transport planning
weight: 60
---

![OSRD presentation](osrd_presentation.png)

OSRD is meant to fulfill a range of use-cases related to railway planning:
 - [timetabling](#timetabling)
 - [operation studies](#%C3%A9tudes-dexploitation)
 - [infrastructure management](#infrastructure-management)
 - [short term capacity management](#short-term-capacity-management)

![produits OSRD](osrd_product.en.png)

## Timetabling

Timetabling is about designing a transport service in order
to meet the transit needs.

OSRD addresses this by providing:

>  - a railway infrastructure database at microscopic scale

Author: microscopic = track level

 - a rolling stock database
 - an infrastructure edition tool

>  - a timetabling tool

Reviewer: "A time table tool" ?

 - train path allowances, including an allowance distribution which minimizes energy use

Planned features include:

>  - automated conflict detection[^conflict]

Reviewer: …and a automatic conflict resolution system?

 - macroscopic infrastructure synthesis from microscopic infrastructure
 - interactive timetable diagrams
 - a throughput and rolling stock turnover optimizer

> [^conflict]: A conflicts occurs when the movement of a train interferes with another

Reviewer: "A conflict" without "s"

## Operation studies

Operation studies assess the ability of infrastructures to meet demand,
and balance infrastructure capacity with transportation needs.
They enable informed public infrastructure investment decisions.

> OSRD addresses this need by providing:

Reviewer: "…addresses these needs…"

 - a railway infrastructure database at microscopic scale
 - a rolling stock database
 - an infrastructure edition tool

>  - a timetabling tool

Reviewer: "time table tool"?

 - train path allowances, including an allowance distribution which minimizes energy use
 - a signaling simulator

Planned features include:

>  - automated conflict detection[^conflict]

Reviewer: …and resolution?

Reviewer 2: Automatic conflict resolution would be quite an ambitious project, isn't it ?

Author: I discussed the matter with Christian, and it seems like there was a misunderstanding on what automatic conflict resolution means. OSRD will support adding a new train while avoiding creating new conflicts, but not automatically solving existing conflicts for an entire timetable. I'll try to make that clearer.

 - automated timetable robustness assessment using stochastic simulation
 - automated throughput analyzer

## Infrastructure management

Processing railway infrastructure data is a requirement shared by all of OSRD's features.
Therefore, a dedicated module provides:
 - a customizable infrastructure map
 - an infrastructure editor
 - a search feature

>  - an import tool[^railjson]

Reviewer: No export tool?

Reviewer 3: Yes of course ! But one day,  in different formats :)

---
title: Use cases
linkTitle: Use cases
description: Use cases related to rail transport planning
weight: 60
---

![OSRD presentation](osrd_presentation.png)

OSRD is meant to fulfill a range of use-cases related to railway planning:
 - [timetabling](#timetabling)
 - [operation studies](#%C3%A9tudes-dexploitation)
 - [infrastructure management](#infrastructure-management)
 - [short term capacity management](#short-term-capacity-management)

![produits OSRD](osrd_product.en.png)

## Timetabling

Timetabling is about designing a transport service in order
to meet the transit needs.

> OSRD addresses this need by providing:

Author: ```suggestion
OSRD addresses this by providing:
```

---
title: Use cases
linkTitle: Use cases
description: Use cases related to rail transport planning
weight: 60
---

![OSRD presentation](osrd_presentation.png)

OSRD is meant to fulfill a range of use-cases related to railway planning:
 - [timetabling](#timetabling)
 - [operation studies](#%C3%A9tudes-dexploitation)
 - [infrastructure management](#infrastructure-management)
 - [short term capacity management](#short-term-capacity-management)

![produits OSRD](osrd_product.en.png)

## Timetabling

> Timetabling is about making an transport offer in response to a
> statement of requirements.

Reviewer 2: I find this sentence a bit weird. Here's my suggestion :

```suggestion
Timetabling is about designing a transport service in order
to meet the transit needs.
```

OSRD addresses this need by providing:

>  - a microscopic railway infrastructure database

Reviewer 2: I get the idea but it can be interpreted like the database is microscopic. Even though it doesn't really make sense the overall bulletpoint ends up rather weird in my opinion. What about something like this ?
```suggestion
 - a railway infrastructure database at microscopic scale
```

 - a rolling stock database
 - an infrastructure edition tool
 - a timetabling tool
 - train path allowances, including an allowance distribution which minimizes energy use

Planned features include:
 - automated conflict detection[^conflict]
 - macroscopic infrastructure synthesis from microscopic infrastructure
 - interactive timetable diagrams
 - a throughput and rolling stock turnover optimizer

[^conflict]: A conflicts occurs when the movement of a train interferes with another

## Operation studies

> Operation studies assess the ability of infrastructure to meet demand,
> and balance infrastructure capacity with transportation needs.
> They enable informed public infrastructure investment decisions.

Reviewer 2: ```suggestion
Operation studies assess the ability of the infrastructure to meet the demand,
and help balancing infrastructure capacity with transportation needs.
They enable informed public infrastructure investment decisions.
```

OSRD addresses this need by providing:

>  - a microscopic railway infrastructure database

Reviewer 2: If we decide to change the corresponding bulletpoint above don't forget to change it here as well.

 - a rolling stock database
 - an infrastructure edition tool
 - a timetabling tool

>  - train path allowances, including an allowance distribution which minimizes energy use

Reviewer 2: Isn't the entire signalling system & station management missing ?

Author: station interlocking is part of conflict detection, I added a line about signaling simulation

Planned features include:
 - automated conflict detection[^conflict]
 - automated timetable robustness assessment using stochastic simulation
 - automated throughput analyzer

## Infrastructure management

Processing railway infrastructure data is a requirement shared by all of OSRD's features.
Therefore, a dedicated module provides:
 - a customizable infrastructure map
 - an infrastructure editor
 - a search feature
 - an import tool[^railjson]

![OSRD management](osrd_management.png)

[^railjson]: OSRD uses the RailJSON rail infrastructure file format.
             It is a free and documented format, designed to be
	     infrastructure manager-agnostic, easy to generate and use.

## Short term capacity management

Even though most train paths[^train-path] are allocated years before operations,
a growing part is allocated shortly before the planned departure time. This change
is largely due to growing demand for rail freight.

There is therefore a need for timely (less than 3 minutes) and correct answers.

A correct train path must, among other things:
 - take planned infrastructure work into account

>  - avoid generating conflicts[^conflict] with already planned train paths,

Reviewer 2: ```suggestion
 - avoid generating conflicts[^conflict] with planned trains,
```
